code/drasil-example/Drasil/PDController/References.hs

```diff
@@ -1,27 +1,12 @@
 module Drasil.PDController.References where
 import Data.Drasil.Citations (smithLai2005)
-import Data.Drasil.People (wikiAuthors)
 
 import Language.Drasil
 
 citations :: BibRef
-citations = [pidWiki, laplaceWiki, johnson2008, abbasi2015, smithLai2005]
+citations = [johnson2008, abbasi2015, smithLai2005]
```

Comment:
So those two citations should still be in the `citations` list here. This is a list of all the citations used in this example, regardless of where they are defined.